go/nogo.rst

```diff
@@ -117,6 +121,54 @@ the ``TOOLS_NOGO`` list of dependencies.
         visibility = ["//visibility:public"],
     )
 
+Usage
+---------------------------------
+
+``nogo`` is invoked automatically when building any Go target in your workspace.
```

Comment:
But it does require setup, right? Should we mention this here?

Reply:
How about 

> ``nogo``, when configured, is invoked automatically when building any Go target in your workspace.

Reply:
Fixed